### ISSUE TITLE
Enhance display 

### ISSUE THREAD

**Issue by llaville** [overtrue/phplint#164]
Hello @overtrue 

I've in mind to propose soon (probably tomorrow) a new version that will : 

- avoid hard-coded values in console input / options

![v60-help](https://user-images.githubusercontent.com/364342/208249886-aca84dc7-fb8d-4593-8dda-8fbd45b17013.png)

- enhance display with PHP runtime / config file used and options like that

![v60-dev](https://user-images.githubusercontent.com/364342/208250066-211a535f-bcc3-469d-aac8-2673c5c44434.png)

Are you agree ?

**Comment by overtrue** [comment]
Wow it's a great idea! Thank you for the contribution.

**Comment by llaville** [comment]
@overtrue All branches are now up-to-date and implement this feature.

Before to release new versions v3, v4, v5 and v6, I want to fix also issue #163.
I'll try to fix it today, but no guarranty

**Comment by llaville** [comment]
@overtrue As I've worked on issue #163, I've re-worked again display improvement especially for options. 
For example, it will give now 

![enhance_display](https://user-images.githubusercontent.com/364342/208909424-c7c21453-726b-4578-aecd-ae12762a915b.png)

**Comment by overtrue** [comment]
It's beautiful!

**Comment by llaville** [comment]
Very few line of code 
```
        if ($verbosity >= OutputInterface::VERBOSITY_DEBUG) {
            $output->writeln('Options       :');
            foreach ($options as $name => $value) {
                $output->writeln(\sprintf("<comment>%13s</comment> > <info>%s</info>", $name, \json_encode($value, \JSON_UNESCAPED_SLASHES)));
            }
        }
```

**Comment by llaville** [comment]
Available in latest releases 3.3.1, 4.4.0, 5.4.1 and 6.0.3
